application-service/src/main/java/ru/art/service/ServiceController.java

```diff
@@ -32,7 +32,7 @@
 import static ru.art.logging.ThreadContextExtensions.*;
 import static ru.art.service.ServiceModule.*;
 import static ru.art.service.ServiceResponseDataExtractor.*;
-import static ru.art.service.constants.RequestValidationPolicy.VALIDATABLE;
+import static ru.art.service.constants.RequestValidationPolicy.*;
```

Comment:
our static analysis engine detects that your pull request includes a change from an explicit import to a wildcard import. According to the best practice (http://www.javadude.com/articles/importondemandisevil.html), import star (import on demand) is a bad practice, wildcard import can sometimes cause name conflicts and ambiguities.